app/bot.py

```diff
@@ -34,36 +33,48 @@
 	# 7:"Thanks anyway!",
 	# 8:"Please fill out this form ",
 
-	10:'Click link below to start conversation ',
-	11:'Please fill out this form ',
-	13:'Thanks anyway!'
+	10: 'Click link below to start conversation ',
```

Comment:
Just calling attention that we may have unintended spacing between the `:` and the following line.

Reply:
To clarify it looks like the linter wants the space. Commented out code should live in a document that isn't the code file.